src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6405,6 +6405,65 @@ export class StudyViewPageStore
         );
     }
 
+    @computed get chartMetaSetForClinicalAttributes(): {
```

Comment:
lets put a comment explaining that this derives the list of columns for the clinical attribute tab from the currently visible charts

Reply:
also can we move all this logic into a helper function to avoid bloating this file any more

Reply:
ditto for the other computed.